core/metadata-3.2.md

```diff
@@ -120,6 +132,23 @@ using `ERR_KEYNOPERMISSION` with an asterisk (`*`) in the `<Key>` field.
 
 *Errors*: `ERR_KEYNOPERMISSION`
 
+### METADATA SYNC
+
+This subcommand requests the full synchronization of metadata associated with
```

Comment:
> I think "requests the full synchronization" doesn't describe this command very well. It looks like it actually means "request the next page of results after getting ERR_METADATASYNCLATER".

It does not mean that, it means what it says.

Or is this meant for when PRIORITY is set to request and clients want to poll all the metadata of a target? Like a bulk get. A "full" synchronization of all keys regardless of priority.

It's a full synchronization of every key that the client is subscribed to.